test/e2e/tests/ppom/ppom-blockaid-alert-simple-send.spec.js

```diff
@@ -106,11 +107,9 @@ async function mockInfuraWithFailedResponses(mockServer) {
   await mockInfura(mockServer);
 
   await mockServer
-    .forPost()
-    .withJsonBodyIncluding({
-      method: 'debug_traceCall',
-      params: [{ accessList: [], data: '0x00000000' }],
-    })
+    .forGet(
+      'https://static.cx.metamask.io/api/v1/confirmations/ppom/ppom_version.json',
+    )
```

Comment:
to cause a warning, it doesn't work anymore the debug_traceCall 500, but we can use this failed request instead